src/main/java/com/marklogic/client/ext/DefaultConfiguredDatabaseClientFactory.java

```diff
@@ -36,7 +36,7 @@ public DatabaseClient newDatabaseClient(DatabaseClientConfig config) {
 			.withSSLHostnameVerifier(config.getSslHostnameVerifier());
 
 		if (config.getSecurityContextType() != null) {
```

Comment:
Unfortunately, the DatabaseClientConfig class in ml-javaclient-util uses "SecurityContextType" (which was created when Authentication was deprecated).